src/views/Amalgamation/ReviewConfirm.vue

```diff
@@ -117,9 +117,15 @@
       >
         <DocumentDelivery
           class="py-8 px-6"
+          :class="{ 'invalid-section': isDocumentDeliveryInvalid }"
```

Comment:
What does this class do, that line 122 doesn't handle?

Reply:
This is the class for invalid section. If remove it, the invalid section will not be highlighted:
![image](https://github.com/user-attachments/assets/7efae118-8d61-4fa7-84c7-a2b8421bc0bd)

Other sections have it too, with it:
![image](https://github.com/user-attachments/assets/3ac3a877-4d28-43d0-ad6e-0a477f3b8083)

Reply:
Looks good. Thanks for the examples!